components/TextInput.js

```diff
@@ -0,0 +1,40 @@
+import propTypes from 'prop-types'
+
+export default function TextInput(props) {
+  return (
+    <div className="mt-6">
+      <label
+        htmlFor={props.id}
+        className="block rounded-t-lg overflow-hidden border-t border-l border-r border-gray-300 px-3 py-2 bg-gray-300 justify-cente"
+      >
+        {props.label}
+      </label>
+      <input
+        type="text"
+        id={props.id}
+        name={props.id}
+        placeholder={props.placeholder}
+        className="appearance-none border-gray-300 border-b border-x w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
```

Comment:
```suggestion
        className="appearance-none border-gray-300 border-b border-x w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:drop-shadow focus:ring-2 focus:ring-inset focus:ring-gray-600"
```
Just a suggestion for a slightly cleaner look when focusing.

Reply:
Added!